internal/cmd/launcher.go

```diff
@@ -249,6 +249,8 @@ func analyze(gs *state.GlobalState, args []string) (k6deps.Dependencies, error)
 	}
 
 	if !isScriptRequired(args) {
+		gs.Logger.
+			Debug("command does not require binary provisioning")
```

Comment:
```suggestion
			Debug("The command to execute does not require binary provisioning")
```
A follow-up improvement might be to return the identified command from `isScriptRequired` function and log it here. However, it can be a low priority improvement for the future.